.travis.yml

```diff
@@ -38,3 +38,13 @@ cache:
   # see https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Caching-with-yarn
     - "frontend/.gradle/yarn"
     - $HOME/.cache/yarn
+
+script:
+  - ./gradlew check
```

Comment:
No need to execute two scripts. Just execute the two tasks at once: `./gradlew check jacocoTestReport`.